Tools/Replay/LR_MsgHandler.h

```diff
@@ -430,9 +432,18 @@ class LR_MsgHandler_NTUN_Copter : public LR_MsgHandler
 class LR_MsgHandler_PARM : public LR_MsgHandler
 {
 public:
-    LR_MsgHandler_PARM(log_Format &_f, DataFlash_Class &_dataflash, uint64_t &_last_timestamp_usec) : LR_MsgHandler(_f, _dataflash, _last_timestamp_usec) {};
+    LR_MsgHandler_PARM(log_Format &_f, DataFlash_Class &_dataflash,
+                       uint64_t &_last_timestamp_usec,
+                       const std::function<bool(const char *name, const float)>&set_parameter_callback) :
+        LR_MsgHandler(_f, _dataflash, _last_timestamp_usec),
+        _set_parameter_callback(set_parameter_callback)
+        {};
 
     virtual void process_message(uint8_t *msg);
+
+private:
+    bool set_parameter(const char *name, const float value);
```

Comment:
You should remove this from LR_MsgHandler right?

Reply:
It's still in there as a wrapper function which filters out sets we never want to do.

Reply:
What am I missing? The `set_parameter` method implementation was changed to belong to `LR_MsgHandler_PARM`, there is no implementation for `LR_MsgHandler`.

Reply:
Ah, gotcha!

Removed old declaration.  And really wished there was an automated way to find these dangling declarations.

Reply:
Yeah, I've tried to find an option for GCC before but I didn't find anything. I guess that the issue is that only on link time it can know if a method implementation is missing or not and, because we remove not referenced methods, no error is produced.